apps/prairielearn/src/lib/workspaceHost.sql

```diff
@@ -259,10 +259,7 @@ WITH
       state = 'terminated',
       terminated_at = NOW()
     WHERE
-      wh.instance_id IN (
-        SELECT
-          UNNEST($instance_ids)
-      )
+      wh.instance_id = ANY ($instance_ids)
```

Comment:
Nit: slap a `::text[]` on here for explicitness and consistency with other queries?

Reply:
Done in 8f7e4438b74d7af63b8d896ff0b39053804f315c.